nautobot/core/templates/generic/object_import.html

```diff
@@ -15,19 +15,22 @@ <h1>{% block title %}{{ obj_type|bettertitle }} Import{% endblock %}</h1>
                 </div>
             </div>
         {% endif %}
-		<form action="" method="post" class="form form-horizontal">
-		    {% csrf_token %}
-		    {% render_form form %}
-            <div class="form-group">
-                <div class="col-md-12 text-right">
-                    <button type="submit" name="_create" class="btn btn-primary">Submit</button>
-                    <button type="submit" name="_addanother" class="btn btn-primary">Submit and Import Another</button>
-		            {% if return_url %}
-                        <a href="{{ return_url }}" class="btn btn-default">Cancel</a>
-                    {% endif %}
+        <form action="" method="post" class="form form-horizontal object-import-form">
+            {% csrf_token %}
+            <div class="panel panel-default">
+                <div class="panel-heading">{{ obj_type|bettertitle }}</div>
+                <div class="panel-body">
+                    {% render_form form %}
```

Comment:
Personally, I'd prefer a vertically-aligned form like the one in my mockup here. This would allow for more visibility of any large data in the textarea. But that would involve creating a new render_form_vertical tag I guess. And might go against the style guide!

![Screenshot 2022-04-22 at 14 11 09](https://user-images.githubusercontent.com/655255/164721253-cf693337-8614-4a53-9694-a032f78376fd.png)

Reply:
Let's see if we can reformat the view to look like this: https://demo.nautobot.com/extras/config-contexts/bcc7f608-e671-403a-8a39-89c2d0a64715/edit/